torch/onnx/__init__.py

```diff
@@ -25,8 +25,12 @@ class CheckerError(Exception):
 
 
 class SymbolicContext:
-    r"""
-    Provides extra context for advanced symbolic functions.
+    r"""Provides extra context for symbolic functions.
+    Args:
+        params_dict (Dict[str, _C.IValue]): Mapping from graph initializer name to IValue.
```

Comment:
Please put the type annotations into the signature of __init__ rather than the doc string. That way they're accessible to `inspect` and mypy.

Reply:
I'm trying to see how to pass the new CI test on doc that was recently added. Saw another example doing it this way, seems like the sphinx standard. But no luck passing it yet.